config/parser.go

```diff
@@ -165,9 +174,19 @@ func (parser ConfigParser) getDefaultConfig() Config {
 				Open:  true,
 				Width: 50,
 			},
-			PrsLimit:    20,
-			IssuesLimit: 20,
-			View:        PRsView,
+			PrsLimit:     20,
+			IssuesLimit:  20,
+			View:         PRsView,
+			DisableViews: []ViewType{},
+			Tabs: []Tab{
```

Comment:
Are you intending to put this here to let the user change the view labels?
I don't really see a use case for that, so I would use a const for that.
I'm writing the default config to the `config.yaml` file if it's the first time the user launches `gh-dash`, so having it here will will also write it to the file.